api/serverless.yml

```diff
@@ -548,3 +548,17 @@ functions:
         handler: src/components/event/routes.dailyEventCheck
         events:
             - schedule: cron(0 8 * * ? *) # Every day at 8 a.m.
+
+    # Sitemaps
+    generateSitemaps:
+        handler: src/components/sitemap/controller.generate
```

Comment:
There's a route.ts file missing under /components/sitemap folder. All the other functions are using (or at least they should use) the methods exported from routes.ts and not from controller.ts directly, for consistency.

Reply:
Ok, have set this up